### PR TITLE
Fix defunct dfm call

### PR DESCRIPTION
**Pull request by kbenoit**
Necessary to unblock CRAN from publishing quanteda v4

### CODE REVIEW

**Comment by juliasilge** [comment]
Thank you so much for this PR @kbenoit! 🙌 

Do I understand correctly that you'd like me to do a tidytext release ASAP?

File: vignettes/tidying_casting.Rmd [hunk 1]
```diff
@@ -88,7 +88,8 @@ Note that a tidier is also available for the `dfm` class from the quanteda packa
 library(methods)
 
 data("data_corpus_inaugural", package = "quanteda")
-d <- quanteda::dfm(data_corpus_inaugural, verbose = FALSE)
+d <- quanteda::tokens(data_corpus_inaugural) %>%
```
Comment (juliasilge) [inline vignettes/tidying_casting.Rmd:91]: I switched to `%>%` because the package as a whole still does. I'll open an issue about that.

**Comment by kbenoit** [comment]
That would be great. Esp since quanteda v4 just got published on CRAN.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by kbenoit** [comment]
Makes perfect sense!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by juliasilge** [comment]
This is failing on R-devel because it can't build the brand new quanteda from source:

> ✖ Failed to build quanteda 4.0.0 (11.6s)

**Comment by kbenoit** [comment]
Yes we are actively trying to fix the r-devel (Fedora) platform build issues. It's tricky but hopefully, won't hold up your update.

**Comment by juliasilge** [comment]
Looks like waiting has only made the problem worse, actually. 😢 Now CI can't build quanteda on R-release either.

@kbenoit do you have any advice on how I can get quanteda up and running in CI? I did get an email from CRAN because of the quanteda release so now tidytext is on a deadline. I can pin an older version of quanteda in CI to get it to pass (right? the older version of quanteda will support the new change in the tidytext vignette?), but of course that defeats the purpose really.

**Comment by kbenoit** [comment]
Resubmitted quanteda today, I think we solved it now. New quanteda.textstats on CRAN, also seededlda and proxyC. 
🤞

note to future self: Don't publish a major release right when the r-devel is also a major release!

**Comment by juliasilge** [comment]
Success now! 🚀 Thank you so much @kbenoit

**Pull request merged**